src/main/scala/chisel3/annotations/annotations.scala

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: Apache-2.0
+
+package chisel3.annotations
+
+import firrtl.annotations._
+import firrtl.FirrtlUserException
+
+/** The lowering convention of a module
+  * Controls how a module's ports are lowered to verilog.
+  */
+sealed class Convention(val string: String) {
+  def serialize: String = string
+}
```

Comment:
implementation based on another annotation in firrtl:
https://github.com/chipsalliance/firrtl/blob/master/src/main/scala/firrtl/annotations/LoadMemoryAnnotation.scala#L11-L26